WordPressComKit/Me.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class Me {
```

Comment:
Could we make models `struct`? If we need Obj-C compatibility, can we at least declare their properties as immutable?
IMO, declaring properties as `var foo: Type!` is a big no-no in Swift, only reserved to cases when we need to interact with existing Obj-C APIs, like `UIViewControllers` "initializing" stuff in `viewDidLoad`